WalletWasabi.Fluent/Models/Wallets/WalletTransactionsModel.cs

```diff
@@ -23,30 +24,31 @@ public partial class WalletTransactionsModel : ReactiveObject
 
 	public WalletTransactionsModel(Wallet wallet)
 	{
+		_wallet = wallet;
+
 		TransactionProcessed =
 			Observable.FromEventPattern<ProcessedResult?>(wallet, nameof(wallet.WalletRelevantTransactionProcessed)).ToSignal()
 					  .Merge(Observable.FromEventPattern(wallet, nameof(wallet.NewFiltersProcessed)).ToSignal())
 					  .Sample(TimeSpan.FromSeconds(1))
 					  .ObserveOn(RxApp.MainThreadScheduler)
 					  .StartWith(Unit.Default);
 
-		List =
+		var transactionChanges =
 			Observable.Defer(() => BuildSummary().ToObservable())
 					  .Concat(TransactionProcessed.SelectMany(_ => BuildSummary()))
 					  .ToObservableChangeSet(x => x.GetHash());
-		_wallet = wallet;
+
+		transactionChanges.Bind(out var collection).Subscribe();
+		Transactions = collection;
```

Comment:
I wouldn't introduce a variable for it as it is not used later. Personal opinion though.

```c#
TransactionProcessed
	.StartWith(Unit.Default)
	.SelectMany(_ => BuildSummary())
	.ToObservableChangeSet(x => x.GetHash())
	.Bind(out var collection)
	.Subscribe();

Transactions = collection;
```